app/views/user/shares/_modal_form.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="modal fade" id="modalSharesForm-<%= document.document_type %>-<%= document.id %>" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
   aria-hidden="true">
-  <div class="modal-dialog" role="document">
+  <div class="modal-dialog modal-lg" role="document">
     <div class="modal-content">
       <div class="modal-header text-center">
-        <h4 class="modal-title w-100">Share</h4>
+        <h4 class="font-weight-bold modal-title w-100">Sharing <%= t("documents.types.#{document.document_type}") %> of <%= document.heading %></h4>
```

Comment:
you could make the `h4.modal-title` have bold weight, instead of adding the extra classes here.

Reply:
:+1: